### ISSUE TITLE
TypeError when try to Device("/dev/sda")

### ISSUE THREAD

**Issue by DGideas** [truenas/py-SMART#32]
I got this error when I try to run `sda = Device("/dev/sda")`:
```python
>>> from pySMART import Device
>>> sda = Device("/dev/sda")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/pySMART/device.py", line 216, in __init__
    cmd = Popen(
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1583, in _execute_child
    and os.path.dirname(executable)
  File "/usr/lib/python3.8/posixpath.py", line 152, in dirname
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

I'm sure that `sda` exists in my `/dev` dir:
```bash
root@my-server:~ # ls /dev | grep sda
sda
sda1
```

So I'm not sure how do I solve this problem. PS, I use Ubuntu 20.04 and Python 3.8.

**Comment by freqlabs** [comment]
It looks like maybe you don't have `smartctl` from `smartmontools` installed?

**Comment by DGideas** [comment]
> It looks like maybe you don't have `smartctl` from `smartmontools` installed?

Yes! It's my fault.

By the way, I make a PR about this problem. I'm not sure if this is appropriate and would appreciate your suggestions on this code. Thanks!

**Comment by XenonOrion** [comment]
I'm seeing the same error, although I have `smartmontools` installed, the interesting thing is that everything runs fine from the terminal, but shows this error when run in `crontab`? Any ideas?

Thanks for this great package, team!

**Comment by DGideas** [comment]
@XenonOrion I think fisrt you have to make sure that `smartmontools` could be accessed via `crontab`.

**Comment by XenonOrion** [comment]
@DGideas , Thanks for the quick reply! You are indeed right, crontab could not access the `smartctl` executable, see [this thread](https://serverfault.com/questions/1032109/cronjob-can-not-execute-smartctl). I've manually added the path `/usr/sbin/` to `$PATH` in the crontab and it works like a charm. Any way, that we can build in a check to see if the executable is accessible?

**Comment by DGideas** [comment]
@XenonOrion From [this question](https://unix.stackexchange.com/questions/401621/where-is-crons-path-set), we know that default `PATH` for cron environment is `/usr/bin:/bin`, so I think binary only in `/usr/bin` and `/bin` could access by `cron`.

**Comment by ralequi** [comment]
> @DGideas , Thanks for the quick reply! You are indeed right, crontab could not access the `smartctl` executable, see [this thread](https://serverfault.com/questions/1032109/cronjob-can-not-execute-smartctl). I've manually added the path `/usr/sbin/` to `$PATH` in the crontab and it works like a charm. Any way, that we can build in a check to see if the executable is accessible?

Probably the check you are asking for have been included on #33 

I'm closing this issue, please, reopen if you need something else
